### PR TITLE
Added Google Drive integration (uploading files)

### PR DESCRIPTION
**Pull request by KrystianDziedziola**

### CODE REVIEW

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/FileMenu.java [hunk 1]
```diff
@@ -134,6 +136,7 @@ private void createMenu()
         this.add(this.fileRecentMenu);
         this.add(this.fileSaveItem);
         this.add(this.fileSaveAsItem);
+        this.add(this.fileSaveToGoogleDrive);
         this.add(this.fileExportMenu);
```
Comment (NdV66) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/FileMenu.java:140]: hmm... why did you "this" use? i know, it's from older code, but maybe we should remove the unnecessary poniter? :) It's only suggestion.

File: violet-framework/src/main/java/com/horstmann/violet/framework/google/drive/GoogleDriveAgent.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ * There is no need to store or ask for credentials, because when you try to authorize,
+ * a web browser will open with a Google Drive website and you will enter your credentials there.
+ */
+public class GoogleDriveAgent
+{
+    private static final String APPLICATION_NAME = "Violet UML Editor";
+    private static final String FILE_TYPE_MIME = "text/html";
+    private static final String CLIENT_SECRETS_RESOURCE_PATH = "/google/drive/client_secrets.json";
+    private static final String USER_ID = "user";
+
```
Comment (NdV66) [inline violet-framework/src/main/java/com/horstmann/violet/framework/google/drive/GoogleDriveAgent.java:35]: why a new line?

**Comment by NdV66** [comment]
please, see my comments, but it looks good :)

**Review by NdV66 — APPROVED** [review]

**Review by abizukojc — DISMISSED** [review]

**Review by abizukojc — COMMENTED** [review]
Why Java 8 ? Actually VioletUML is in Java 6.

**Pull request merged**